flink-cdc-common/src/main/java/com/ververica/cdc/common/event/Event.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2023 Ververica Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ververica.cdc.common.event;
+
+import java.util.Date;
+
+/**
+ * Class {@code Event} is the super interface defines the events of external systems flowing into
+ * Flink CDC.
+ */
+public interface Event {
+    /** Describes the actual occurrence time of the event. */
+    Date eventTime();
```

Comment:
The `Event` should be a mark interface,  eventTime likes strange here as we can obtain the information like partition,op_ts from metadata fields.